src/ResponseTagger.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/*
+ * This file is part of the FOSHttpCache package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\HttpCache;
+
+use FOS\HttpCache\ProxyClient\Invalidation\TagsInterface;
+
+/**
+ * Service for Response cache tagging.
+ *
+ * @author David de Boer <david@driebit.nl>
+ * @author David Buchmann <mail@davidbu.ch>
+ * @author André Rømcke <ar@ez.no>
+ */
+class ResponseTagger
+{
+    /**
+     * @var TagsInterface
+     */
+    private $client;
+
+    /**
+     * @var array
+     */
+    private $tags = [];
+
+    /**
+     * Constructor
+     *
+     * @param TagsInterface $client
+     */
+    public function __construct(TagsInterface $client)
+    {
+        $this->client = $client;
+    }
+
+    /**
+     * Get the HTTP header name that will hold cache tags.
+     *
+     * @return string
+     */
+    public function getTagsHeaderName()
+    {
+        return $this->client->getTagsHeaderName();
```

Comment:
lets keep it like this. we want to have a default header name and if there are several places, users that do not use the bundle have a dangerous live.